src/table/virtualized-table/hooks/use-dynamic-row-height.ts

```diff
@@ -84,7 +78,14 @@ const useDynamicRowHeight = ({
 
   const setCellSize = useCallback(
     (text: string, rowIdx: number, colIdx: number) => {
+      const key = `${rowIdx}-${colIdx}`;
+      if (rowMeta.current.measuredCells.has(key)) {
```

Comment:
Do you need this guard while using Set? is it for avoiding replacement of keys old value?

Reply:
I was asking because you've used Set for `measuredCells` and since that would not store duplicate values then thinking that check might be not needed, but since it is for preventing re-calc then it's great to have that check ✅